regression/python/import-as/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.py
+
+^VERIFICATION SUCCESSFUL$
```

Comment:
Can you also add a test case that provides `VERIFICATION FAILED`? For example, you could check for overflow.